Automation.sc

```diff
@@ -309,14 +310,15 @@ Automation {
 
 
     // internal constructor function
-    constructor { |ilength, iserver, ishowLoadSave|
+    constructor { |ilength, iserver, ishowLoadSave, iminTimeStep|
         // evaluate input args
 
         server = iserver;
         length = 0.0 + ilength; // make sure it is a float
 
         showLoadSave = ishowLoadSave;
-
+		minTimeStep = iminTimeStep;
+		
```

Comment:
and here you're changing a blank line that has no spaces to a blank line that has many spaces (rolling eyes) ;)